judge/models/tests/test_problem.py

```diff
@@ -13,6 +13,14 @@ class ProblemTestCase(CommonDataMixin, TestCase):
     def setUpTestData(self):
         super().setUpTestData()
 
+        self.users.update({
```

Comment:
Note to reviewer: I kept this user here instead of in `tests/util.py` where all the other problem permission related users are because I felt this user wouldn't be very useful outside of this specific testcase. Should it be moved with the other problem related users?